azure-devops/azext_devops/test/team/test_user.py

```diff
@@ -0,0 +1,103 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+
+try:
+    # Attempt to load mock (works on Python 3.3 and above)
+    from unittest.mock import patch
+except ImportError:
+    # Attempt to load mock (works on Python version below 3.3)
+    from mock import patch
+
+from azext_devops.vstsCompressed.member_entitlement_management.v4_1.member_entitlement_management_client import (MemberEntitlementManagementClient)
+from azext_devops.dev.team.user import (get_user_entitlements,
+                                        get_user_entitlement,
+                                        add_user_entitlement,
+                                        delete_user_entitlement,
+                                        update_user_entitlement)
+
+from azext_devops.dev.common.services import clear_connection_cache
+
+    
+class TestUserMethods(unittest.TestCase):
```

Comment:
Check the change in this PR to refactor authentication mocks into a base class. https://github.com/Microsoft/azure-devops-cli-extension/pull/469

Can you move this to common place and consume in your tests.

Reply:
Sure, will refactor this along with Group management commands /tests.